oeps/oep-0044-bp-changelogs.rst

```diff
@@ -0,0 +1,294 @@
+
+The recommendation also includes "**Security** in case of vulnerabilities", but that is being called out separately because the Security Group needs to manage the timing of adding any such designation to any public changelog.
+
+Other Recommendations
+~~~~~~~~~~~~~~~~~~~~~
+
+* Format dates as YYYY-MM-DD. As an example, 2020-03-21.
+* If you do nothing else, list deprecations, removals, and any breaking changes in your changelog.
+* It is both acceptable and appropriate to clean up changelog entries at any time to add clarifications or customize messaging for the audience of the changelog when needed.
+
+Example Changelog
+~~~~~~~~~~~~~~~~~
+
+Here is an example CHANGELOG.rst.
+
+.. note:: Although this example includes versions, based on the decisions below our changelog entries would typically only include dates, because our versioned changelogs are managed using GitHub Releases.
+
+::
+
+  Changelog
+  ===========
+
+  All notable changes to this project will be documented in this file.
+
+  The format is based on recommendations in `OEP-44`_.
+  This project adheres to `Semantic Versioning`_.
+
+  .. _OEP-44: https://open-edx-proposals.readthedocs.io/en/latest/oep-0044-bp-changelogs.html
+  .. _Semantic Versioning: https://semver.org/spec/v2.0.0.html
+
+  [2.0.0] - 2020-03-21
+  --------------------
+
+  Removed
+  ~~~~~~~
+
+  * Removed deprecated function `xxx`. See DEPR-XXX for details.
+
+  [2.0.0] - 2020-03-21
+  --------------------
+
+  Added
+  ~~~~~
+
+  * Feature x. See announcement here: https://xxx
+
+Notable Differences
+~~~~~~~~~~~~~~~~~~~
+
+The following are notable differences between the recommendations of this OEP and the `keepachangelog.com`_ recommendations:
+
+* We use GitHub Releases in certain cases, as you'll see below, for developer efficiency purposes.
+* Because we typically release every change for libraries, we don't get any benefit from an "Unreleased" section.
+* We sometimes use automation that uses git commit messages for changelog messages. As noted above, it is acceptable and appropriate to clean these up as needed.
+
+Library Repositories
+--------------------
+
+All Open edX library repositories should contain a CHANGELOG.rst. These libraries should all adhere to `Semantic Versioning`_.
+
+Semantic Versioning
+~~~~~~~~~~~~~~~~~~~
+
+All versioned code should use a MAJOR.MINOR.PATCH format for versioning, as defined by the `Semantic Versioning v2`_ specification. Here is a copy of the most relevant part of the summary:
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+* *MAJOR* version when you make *incompatible API changes*,
+* *MINOR* version when you *add functionality in a backwards compatible manner*, and
+* *PATCH* version when you make *backwards compatible bug fixes*.
+
+Every published version should have a corresponding entry in the changelog, but not necessarily in CHANGELOG.rst.
```

Comment:
I don't know what this means.  How are "the changelog" and "CHANGELOG.rst" different?

Reply:
I made this more clear.